lib/models/installation.js

```diff
@@ -28,21 +28,31 @@ module.exports = (sequelize, DataTypes) => {
     },
 
     async sync(github, { owner, repo }) {
-      const { data } = await github.request({
-        method: 'GET',
-        url: '/repos/:owner/:repo/installation',
-        headers: {
-          accept: 'application/vnd.github.machine-man-preview+json',
-        },
-        owner,
-        repo,
-      });
-
+      const options = repo
+        ? {
+          method: 'GET',
+          url: '/repos/:owner/:repo/installation',
+          headers: {
+            accept: 'application/vnd.github.machine-man-preview+json',
+          },
+          owner,
+          repo,
+        }
+        : {
+          method: 'GET',
+          url: '/users/:username/installation',
+          headers: {
+            accept: 'application/vnd.github.machine-man-preview+json',
+          },
+          username: owner,
+        };
```

Comment:
GitHub Apps Question: If someone has installed the "Slack" GitHub App on a single repository, then will making an API request  to `/users/:username/installation` tell us about whether it's installed on the whole org or just on that one repo?